core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1091,10 +1091,11 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
       }
       // load metadata for 1 level of descendants, or all descendants if recursive
       LoadMetadataContext loadMetadataContext = LoadMetadataContext.mergeFrom(
-          LoadMetadataPOptions.newBuilder().setCreateAncestors(true)
+          LoadMetadataPOptions.newBuilder()
```

Comment:
actually no change, just the format?

Reply:
If yes, I'd suggest not touching this file

Reply:
yeah this changes should not be in, let me revert it